db/migrate/20210312074713_add_provider_inputs.rb

```diff
@@ -0,0 +1,10 @@
+class AddProviderInputs < ActiveRecord::Migration[6.0]
+  def change
+    create_table :template_invocation_provider_input_values do |t|
+      t.references :template_invocation, :null => false, :index => { :name => 'idx_templ_inv_provider_input_values_on_templ_inv_id' }
```

Comment:
why are we naming the index? why not just `index: true`?

Reply:
Because autogenerated index name is too long and it crashes the migration.

Reply:
would simplier name of the table help ? :thinking: